config/src/main/java/com/quorum/tessera/config/util/EnvironmentVariables.java

```diff
@@ -0,0 +1,14 @@
+package com.quorum.tessera.config.util;
+
+public final class EnvironmentVariables {
+
+    public static final String serverKeyStorePwd = "TESSERA_SERVER_KEYSTORE_PWD";
```

Comment:
java naming convention is static final in uppercase with underscore word separators.